src/components/Shared/FeedbackDialog.vue

```diff
@@ -1,14 +1,17 @@
 <template>
   <q-card style="min-width: 350px" class="no-padding">
       <modal-header>Feedback</modal-header>
-
         <q-card-section>
           <div class="">Please tell us what is on your mind. Try to describe the feedback as detailed as possible.</div>
           <q-banner v-if="!loggedIn" rounded class="bg-orange-2 text-orange-10">You're not logged in. You can still write some feedback for us, but we won't be able to respond to you. If you want to have a response, please log in first.</q-banner>
         </q-card-section>
+        <q-card-section class="bg-blue-3 q-ma-md" v-if="admin">
+          <div>Only visible for admins!</div>
```

Comment:
Is this still needed or is this leftover debug code?

Reply:
This is new extra information that will show more information regarding the device data of the user who send feedback.